### ISSUE TITLE
pdf2htmlEX's implementation of Ligatures do not work in the Blink/Webkit browser engines

### ISSUE THREAD

**Issue by stephengaito** [pdf2htmlEX/pdf2htmlEX#68]
pdf2htmlEX's implementation of Ligatures do not work in the Blink/Webkit browser engines

See: https://github.com/pdf2htmlEX/pdf2htmlEX/issues/57#issuecomment-650557076

**Comment by stephengaito** [comment]
Correction.... By default Gecko(FireFox) seems to have ligatures turned off while Blink and Webkit have ligatures turned on.

Unfortunately Blick(Chrome) and Webkit(Safari) respond to different css to turn ligatures off:

@font-face{.... font-feature-settings: "liga" 0, "clig" 0; } works in Gecko, Blink and Webkit (but is "low-level")

@font-face{.... font-variant-ligatures:none; }  works in Gecko and Blink (but does not work in Webkit)

@font-face{.... } works in Gecko (since ligatures are disabled by default)

See: http://blog.fonts.com/2016/06/30/to-ligature-or-not-to-ligature/

**Comment by stephengaito** [comment]
**SOLUTION:** 

Add a new command line switch `--turn-off-ligatures <int>` with a value of 

- `0` (no explicit css to turn off ligatures) or 
- `1` (emit explicit css to turn off ligatures). 

The default will be `0`.

When `--turn-off-ligatures=1` we will add
```
 font-variant-ligatures:none;font-feature-settings: "liga" 0, "clig" 0, "dlig" 0, "hlig" 0, "calt" 0;
```
to all `@font-face` definitions generated in the file specific `*.css` file.

**Comment by stephengaito** [comment]
Need to edit:

1. src/pdf2htmlEX.cc
2.src/HTMLRenderer/font.cc (between lines 1036 and 1058)
3. pdf2htmlEX.1.in

Then TEST TEST TEST

**Comment by stephengaito** [comment]
Fixed in 0.18.8.rc1
